synthesizer/program/src/traits/stack_and_registers.rs

```diff
@@ -98,6 +98,9 @@ pub trait StackProgram<N: Network> {
     /// Returns the program checksum.
     fn program_checksum(&self) -> &[U8<N>; 32];
 
+    /// Returns the call graph checksum for a function, if it exists.
+    fn call_graph_checksum(&self, function_name: &Identifier<N>) -> Result<Option<Field<N>>>;
```

Comment:
Is there an opportunity to unify `call_graph_checksum` and `compute_call_graph_checksum`?

Reply:
That was my initial implementation, it was harder to read and name for not a huge performance win.